concent_api/api-e2e-cluster-multiple-golem-messages-routing.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+from golem_messages.message import Ping
+
+from api_testing_common import api_request
+from api_testing_common import count_fails
+from api_testing_common import PROVIDER_PRIVATE_KEY
+from api_testing_common import run_tests
+from protocol_constants import ProtocolConstants
+
+import requests
+
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
+
+
+@count_fails
+def test_case_ping_message_send_to_concent_with_supported_golem_messages_should_get_http400_response_with_message_unknown_error(
```

Comment:
Maybe we should write a short multi line comment (each line starting with #) why we expect 400 for supported version? It might be confusing for someone not really familiar with the case. The text can be copied from the issue (with little modification).

Reply:
Added